functions.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var numOfGallons = function(MPG,distance){
```

Comment:
Better naming: calculateNumberOfGallons,
Also mpg not MPG  - a bit of a weird case this one.